--- h2database-1.4.187/h2/src/main/org/h2/jdbc/JdbcDatabaseMetaData.java	2015-04-10 14:14:31.000000000 -0430
+++ h2database-1.4.187~/h2/src/main/org/h2/jdbc/JdbcDatabaseMetaData.java	2015-07-02 09:31:15.853056817 -0430
@@ -3131,12 +3131,9 @@
     /**
      * [Not supported]
      */
-    /*## Java 1.7 ##
-    @Override
     public boolean generatedKeyAlwaysReturned() {
         return true;
     }
-    //*/
 
     /**
      * [Not supported]
@@ -3149,13 +3147,10 @@
      * @param columnNamePattern null (to get all objects) or a column name
      *            (uppercase for unquoted names)
      */
-    /*## Java 1.7 ##
-    @Override
     public ResultSet getPseudoColumns(String catalog, String schemaPattern,
             String tableNamePattern, String columnNamePattern) {
         return null;
     }
-    //*/
 
     /**
      * INTERNAL
