--- h2database-1.4.187/h2/src/main/org/h2/jdbc/JdbcStatement.java	2015-04-10 14:14:31.000000000 -0430
+++ h2database-1.4.187~/h2/src/main/org/h2/jdbc/JdbcStatement.java	2015-06-21 14:36:17.802023113 -0430
@@ -929,22 +929,20 @@
     /**
      * [Not supported]
      */
-/*## Java 1.7 ##
+
     @Override
     public void closeOnCompletion() {
         // not supported
     }
-//*/
 
     /**
      * [Not supported]
      */
-/*## Java 1.7 ##
+
     @Override
     public boolean isCloseOnCompletion() {
         return true;
     }
-//*/
 
     // =============================================================
 
